demos/button/icons.html

```diff
@@ -4,44 +4,59 @@
 	<meta charset="utf-8">
 	<title>jQuery UI Button - Icons</title>
 	<link rel="stylesheet" href="../../themes/base/all.css">
-	<script src="../../external/jquery/jquery.js"></script>
-	<script src="../../ui/core.js"></script>
-	<script src="../../ui/widget.js"></script>
-	<script src="../../ui/button.js"></script>
 	<link rel="stylesheet" href="../demos.css">
-	<script>
-	$(function() {
-		$( "button:first" ).button({
-			icons: {
-				primary: "ui-icon-locked"
-			},
-			text: false
-		}).next().button({
-			icons: {
-				primary: "ui-icon-locked"
-			}
-		}).next().button({
-			icons: {
-				primary: "ui-icon-gear",
-				secondary: "ui-icon-triangle-1-s"
-			}
-		}).next().button({
-			icons: {
-				primary: "ui-icon-gear",
-				secondary: "ui-icon-triangle-1-s"
-			},
-			text: false
-		});
-	});
+	<script src="../../external/requirejs/require.js"></script>
+	<script src="../bootstrap.js">
+		$( ".widget button" ).first()
+			.button()
+			.next().button( {
+				icon: "ui-icon-gear",
+				showLabel: false
+			} ).next().button( {
+				icon: "ui-icon-gear"
+			} ).next().button( {
+				icon: "ui-icon-gear",
+				iconPosition: "end"
+			} ).next().button( {
+				icon: "ui-icon-gear",
+				iconPosition: "top"
+			} ).next().button( {
+				icon: "ui-icon-gear",
+				iconPosition: "bottom"
+			} );
```

Comment:
I know the style was already like this, but let's make this follow our actual indentation rules. It's probably a lot easier if you use `.eq(x)` and `.end()` since the current form would just keep moving further in.